cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java

```diff
@@ -249,18 +195,19 @@ private boolean setRtcRegulatingControlVoltage(String rtcId, boolean regulating,
             return false;
         }
 
-        // Even if regulating is false, we reset the target voltage if it is not valid
-        if (control.targetValue <= 0) {
-            context.ignored(rtcId,
+        // We always keep the targetValue
+        // It targetValue is not valid, emit a warning and deactivate regulating control
+        boolean validTargetValue = control.targetValue > 0;
+        if (!validTargetValue) {
```

Comment:
In this case, maybe set targetValue to 0? I'm not sure if ValidationUtil will not throw an exception otherwise (and specify in the log)

Reply:
The validation only checks targetV if it is enabled. In this case the control is deactivated.